apps/perf-test/src/scenarios/MakeStyles.tsx

```diff
@@ -66,7 +66,7 @@ const staticViewStyles: any = [
   ],
 ];
 
-const useStaticViewStyles = makeStyles(staticViewStyles);
+const useStaticViewStyles = makeStylesCompat(staticViewStyles);
```

Comment:
you should create a new function, since this change will break consumers of `makeStyles` not in the monorepo

Although I don't know how important that is :/